### PR TITLE
Show connection type in connection tracker

### PR DESCRIPTION
**Pull request by gzdunek**
image

### CODE REVIEW

File: packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx [hunk 1]
```diff
@@ -100,3 +112,18 @@ export function ConnectionItem(props: ConnectionItemProps) {
     </ListItem>
   );
 }
+
+function getKindName(kind: ExtendedTrackedConnection['kind']): string {
+  function assertUnreachable(x: never): never {
+    throw new Error(`Unhandled case: ${x}`);
+  }
```
Comment (ravicious) [inline packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx:119]: Could you move this out of `getKindName`? :D

File: packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx [hunk 2]
```diff
@@ -75,7 +75,19 @@ export function ConnectionItem(props: ConnectionItemProps) {
...
+                background: rgba(255, 255, 255, 0.25);
+                opacity: 0.85;
+                padding: 1px 2px;
+                margin-right: 4px;
+                border-radius: 4px;
+              `}
+            >
+              {getKindName(props.item.kind)}
+            </span>
+            <span>{props.item.title}</span>
```
Comment (ravicious) [inline packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx:90]: Could you add `vertical-align: middle` to the span with connection title? It should align those two spans that have different font sizes.

**Review by ravicious — APPROVED** [review]
Tested with long resource name and it appears to work just fine.

File: packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx (hunk 1 above)
Comment (gzdunek) [reply packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx]: Done

File: packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx (hunk 2 above)
Comment (gzdunek) [reply packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx]: Done 👍

**Pull request merged**